changelogs/unreleased/gh-8096-sql-seq-scan-default.md

```diff
@@ -0,0 +1,3 @@
+## feature/compat
+
+* Compat option sql_seq_scan_default was introduced (gh-8096).
```

Comment:
The SEQSCAN feature was never released so we shouldn't add a separate changelog entry. Also, there's no such section 'feature/compat'. You should update the original changelog entry for SEQSACAN instead. PTAL at how other compat options are described in changelogs.

Also, the doc bot request isn't required, since you've already documented it [here](https://github.com/tarantool/tarantool/wiki/compat%3Asql_seq_scan_default).

Reply:
Reminder for this particular option? What about other compat options? I think it's better to create a separate ticket for this task manually.

Reply:
Removed doc-request and added `NO_DOC`.

Reply:
As for `feature/compat` - I saw commit headers that started with `compat:`, so I thought `compat` was the new section.